faq/serverless-jobs.mdx

```diff
@@ -84,19 +84,18 @@ Serverless Jobs are integrated with [Cockpit](/observability/cockpit/quickstart/
 
 ## Can I cancel or modify a Serverless Job after it has started?
 
-An ongoing Serverless Job can be interrupted during its execution from the **Job runs** section of a job's **Overview** tab.
+An ongoing Serverless Job can be interrupted during its execution from the **Job runs** section of a job's **Overview** tab. Refer to the [dedicated docmentation](/serverless/jobs/how-to/stop-job/) for more information.
 
 ## How can I automate the deployment and management of Scaleway Serverless Jobs?
 
 Scaleway Serverless Jobs is part of the Scaleway ecosystem, it can therefore be driven using the [Scaleway CLI](/developer-tools/scaleway-cli/quickstart/), the [Scaleway API](https://www.scaleway.com/en/developers/api/), and other [developer tools](https://www.scaleway.com/en/developers/). Our serverless ecosystem offers a lot of possibilities with event-driven architectures, and integrations with more products of the Scaleway ecosystem are under active development.
 
-## Do Serverless Jobs offer parallelization?
+## Can I run multiple Serverless Jobs at the same time?
 
-Not yet. Scaleway Serverless Jobs will soon offer parallelization via a `parallelism` parameter for each job. This will automatically launch a given number of replicas of the Job container in parallel.
+When starting a Job, you can use contextual options to define the number of jobs to execute at the same time. Refer to the [dedicated documentation](/serverless/jobs/how-to/run-job/#how-to-run-a-job-with-contextual-options) for more information.
```

Comment:
not sure about capitalization of job at start of sentence and then not at the second mention...but it's a minefield...